orttraining/README.md

```diff
@@ -0,0 +1,83 @@
+# Introduction
+
+ONNX Runtime Trainer is a test feature introduced in the ONNX Runtime engine. This trainer can be used to accelerate the computation of the ops used to train transformer class models.
+
+The ONNX Runtime trainer can be used with your existing Pytorch training code to accelerate execution on NVIDIA GPU clusters.
+
+## Build on Linux
+
+Build the ONNX Runtime Training engine from source to use with NVIDIA GPUs for accelerating the computations.
+
+### Dependencies
+
+This default NVIDIA GPU build requires CUDA runtime libraries installed on the system:
+
+* The GPU-accelerated CUDA libraries [CUDA 10.1](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1604&target_type=debnetwork)
+
+* The GPU-accelerated library of primitives for deep neural networks [cuDNN 7.6.2](https://docs.nvidia.com/deeplearning/sdk/cudnn-install/index.html#install-linux)
+
+* The NVIDIA Collective Communications Library (NCCL) multi-GPU and multi-node communication primitives library [NCCL2](https://docs.nvidia.com/deeplearning/sdk/nccl-install-guide/index.html)
+
+* OpenMPI 4.0.0.0
+
+```
+wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.0.tar.gz
+tar zxf openmpi-4.0.0.tar.gz
+cd openmpi-4.0.0
+./configure --enable-orterun-prefix-by-default
+make -j $(nproc) all
+sudo make install
+sudo ldconfig
+```
+
+### Get the code and setup the environment
+
+* Checkout this code repo with `git clone https://github.com/microsoft/onnxruntime`
+
+* Set the environment variables: __adjust the path for location your build machine__
+
+```
+export CUDA_HOME=<location for CUDA libs> # e.g. /usr/local/cuda
+export CUDNN_HOME=<location for cuDNN libs> # e.g. /usr/local/cuda
+export CUDACXX=<location for NVCC> #e.g. /usr/local/cuda/bin/nvcc
+export PATH=<location for openmpi/bin/>:$PATH
+export LD_LIBRARY_PATH=<location for openmpi/lib/>:$LD_LIBRARY_PATH
+export MPI_CXX_INCLUDE_PATH=<location for openmpi/include/>
+source /data/intel/impi/2018.3.222/intel64/bin/mpivars.sh
+```
+
+### Create the ONNX Runtime wheel
+
+Run `./build.sh --enable_training --use_cuda --config=RelWithDebInfo --build_wheel`

```

Comment:
maybe explicitly specify that the user's current directory should be the onnxruntime repo.

Reply:
Added a step to cd onnxruntime.